VisualizationBase/src/overlays/ArrowOverlay.cpp

```diff
@@ -79,4 +96,22 @@ void ArrowOverlay::updateGeometry(int, int)
 					 second->scenePos().y() + second->heightInScene() / 2 - leftTopCorner.y()}.toPoint();
 }
 
+void ArrowOverlay::hoverEnterEvent(QGraphicsSceneHoverEvent *) {
```

Comment:
I know this handling code is minimal, but you should still put it in a corresponding `HArrowHandler` class.